RUFAS/output_manager.py

```diff
@@ -233,6 +238,28 @@ def add_variable(self, name: str, value: Any, info_map: Dict[str, Any], first_in
             for k, v in value.items():
                 self._variables_usage_counter[f"{key}.{k}"] = 0
 
+    def _save_current_variable_pool(self) -> None:
+        """
+        Save the current variable pool into JSON file. Flush the variable pool and reset the pool size.
+        """
+        info_map = {
+            "class": self.__class__.__name__,
+            "function": self._save_current_variable_pool.__name__,
+        }
+
+        self.create_directory(self.saved_pool_chunks_path)
+        saved_pool_file_name = self.generate_file_name(f"saved_pool_{self.saved_pool_chunks_num}", "json")
+        saved_pool_file_path = Path.joinpath(self.saved_pool_chunks_path, saved_pool_file_name)
+        self.dict_to_file_json(data_dict=self.variables_pool, path=saved_pool_file_path, minify_output_file=False)
```

Comment:
```suggestion
        self.dict_to_file_json(data_dict=self.variables_pool, path=saved_pool_file_path, minify_output_file=True)
```
Would minifying these files be better? It would mean less space taken up by the files, and might help save a little time reloading them.